libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -348,7 +359,8 @@ bool AP_ICEngine::engine_control(float start_control, float cold_start, float he
     RC_Channel *c = rc().channel(start_chan-1);
     if (c != nullptr) {
         // get starter control channel
-        if (c->get_radio_in() <= 1300) {
+        uint16_t cvalue = c->get_radio_in();
+        if (cvalue >= start_chan_min_pwm && cvalue <= 1300) {
```

Comment:
Other than zero check, what is the use-case for this? Seems unnecessary to add a param. Maybe change it to hardcoded 900-1300?

Reply:
it is very useful, as the RFX900x defaults to sending 1000 when you lose the link. For those flying with PPM over the RFD900x this parameter can save the aircraft